File: kftw2/android/src/main/java/com/lavaeater/kftw/android/AndroidLauncher.java
package com.lavaeater.kftw.android;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
import com.lavaeater.kftw.KidsFromTheWastelandGame;

import data.GameSettings;

/** Launches the Android application. */
public class AndroidLauncher extends AndroidApplication {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        AndroidApplicationConfiguration configuration = new AndroidApplicationConfiguration();
        initialize(new KidsFromTheWastelandGame(new GameSettings(144f, 112f, 8, 20, "sv")), configuration);
    }
}

File: kftw3/android/src/main/kotlin/core/android/AndroidLauncher.kt
package core.android

import android.os.Bundle

import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration
import KidsFromTheWasteland

/** Launches the Android application. */
class AndroidLauncher : AndroidApplication() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        initialize(KidsFromTheWasteland(), AndroidApplicationConfiguration())
    }
}
